sources/api/schnauzer/src/helpers.rs

```diff
@@ -975,6 +987,47 @@ pub fn join_array(
     Ok(())
 }
 
+/// `toml_encode` accepts arbitrary input and encodes it as a toml string
+///
+/// # Example
```

Comment:
If this is a separate commit in the core kit, you should be able to `git cherry-pick -x` it here so it's clear where it originated and that it's the same code change.

Reply:
As discussed, separated the commit. Cherry pick won't work due to the schnauzer crate diverging in bob and core-kit.